### PR TITLE
feat: remove identity certificate

### PR DESCRIPTION
**Pull request by andrewrynhard**
Signed-off-by: Andrew Rynhard <andrew@andrewrynhard.com>

### CODE REVIEW

**Comment by andrewrynhard** [comment]
@TimJones PTAL

**Comment by TimJones** [comment]
@andrewrynhard Looks good, but maybe we should replace the `talos_identity_ip_addresses` variable with something like `talos_target` since now the only reference is for use in the `admin_config` template.
It would allow for flexibility in using IP or DNS as a target without forcing either one.

File: variables.tf [hunk 1]
```diff
@@ -1,6 +1,5 @@
-variable "talos_identity_ip_addresses" {
-  description = "A list of the master node IP addresses."
-  type        = "list"
+variable "talos_target" {
```
Comment (TimJones) [inline variables.tf]: Should `type` be explicitly defined as `string`?

**Comment by TimJones** [comment]
@andrewrynhard LGMT :+1:

File: variables.tf (hunk 1 above)
Comment (andrewrynhard) [reply variables.tf]: I always favor explicit over implicit. The default is `string`, but wouldn't hurt to be explicit.

**Pull request merged**